packages/fxa-auth-server/lib/luaScripts/setAccessToken_2.lua

```diff
@@ -1,11 +1,17 @@
 local userId = KEYS[1]
 local tokenId = KEYS[2]
 local token = ARGV[1]
-local ttl = ARGV[2]
+local limit = tonumber(ARGV[2])
+local ttl = tonumber(ARGV[3])
 local args = {token}
 if ttl then
   table.insert(args, 'PX')
   table.insert(args, ttl)
 end
+local count = redis.call('scard', userId)
+if count > limit then
+  redis.call('spop', userId, math.min((count - limit) + math.floor(limit / 10), limit))
+end
 redis.call('set', tokenId, unpack(args))
 redis.call('sadd', userId, tokenId)
+redis.call('expire', userId, math.max(ttl, 259200))
```

Comment:
Can you add a comment to call out what the magic number is?

(The intent is to eventually lower this TTL, and if so, max 3 days is still okay. But if we went in the other direction for some unforeseen reason, this would be wrong. Maybe pass in from the config?)

Reply:
yeah, i was trying to avoid piping that config variable through. I'll add a note